owners.json

```diff
@@ -1,18 +1,9 @@
 {
-  "amitaekbote": {"slack": "amitaekbote", "jira": "aekbote"},
-  "adam-mesos": {"slack": "adam", "jira": "adam"},
-  "asridharan ": {"slack": "avinash", "jira": "avinash"},
   "branden": {"slack": "branden", "jira": "branden"},
   "gpaul": {"slack": "gpaul", "jira": "gpaul"},
   "jgehrcke": {"slack": "jp", "jira": "jp"},
   "klueska": {"slack": "klueska", "jira": "klueska"},
-  "lingmann": {"slack": "jeremy", "jira": "jeremy"},
-  "MatApple": {"slack": "mat", "jira": "mat"},
-  "mnaboka": {"slack": "mnaboka", "jira": "mnaboka"},
-  "mellenburg": {"slack": "mellenburg", "jira": "michael.ellenburg"},
   "meichstedt": {"slack": "matthias.eichstedt", "jira": "matthias.eichstedt"},
   "orlandohohmeier": {"slack": "orlando", "jira": "orlando"},
   "orsenthil" : {"slack": "skumaran", "jira": "skumaran"},
-  "spahl": {"slack": "seb", "jira": "sebastien"},
-  "vespian": {"slack": "prozlach", "jira": "prozlach"}
```

Comment:
For the curious - @vespian is keeping rough company in this commit, but AFAIK just doesn't want to be in this list anymore.

Reply:
He removed himself from the owners.json in the master branch but forgot to backport https://github.com/dcos/dcos/pull/2836